recipes/default.rb

```diff
@@ -916,7 +918,7 @@
 directory node['hopsworks']['staging_dir'] + "/private_dirs"  do
   owner node['jupyter']['user']
   group node['hopsworks']['group']
-  mode "0330"
+  mode "0370"
```

Comment:
@kouzant Beamjobserver filebeat runs as glassfish user and needs to access all secret dirs, hence we give read permission. Is that ok?

Reply:
As long as we never add `jupyter` user to `glassfish` group, it's ok